htdocs/projet/class/project.class.php

```diff
@@ -805,8 +805,8 @@ public function get_element_list($type, $tablename, $datefieldname = '', $date_s
 			$sql = "SELECT DISTINCT pt.rowid FROM ".MAIN_DB_PREFIX."projet_task as pt WHERE pt.fk_projet IN (".$this->db->sanitize($ids).")";
 		} elseif ($type == 'element_time') {	// Case we want to duplicate line foreach user
 			$sql = "SELECT DISTINCT pt.rowid, ptt.fk_user FROM ".MAIN_DB_PREFIX."projet_task as pt, ".MAIN_DB_PREFIX."element_time as ptt WHERE pt.rowid = ptt.fk_element AND ptt.elementtype = 'task' AND pt.fk_projet IN (".$this->db->sanitize($ids).")";
-		} elseif ($type == 'stock_mouvement') {
```

Comment:
Why removing this case ? Can't you add the new case without removing this one ?

Reply:
It's my understanding from looking around that the stock_mouvement class in the Projects is depreciated for the following reasons:

1. Already, the only way to display stock transfers (of either class) is to set the undocumented global config variable STOCK_MOVEMENT_INTO_PROJECT_OVERVIEW. Once it has been set, nothing from stock_mouvement class can displays anyway, because:
2. To the best of my knowledge the stock movements in stock_mouvement does not have any way to associate with project in a stock Dolibarr install. It appears that this functionality has been moved into only advanced stock transfer unless manually adding extrafields/complementary attributes to stock_mouvment.

If I am wrong (or at least we cannot be certain that I am correct), I will happily make the alterations just-in-case, but I'm not sure if it makes sense to leave the old case if its depreciated.